src/citrine/resources/predictor.py

```diff
@@ -81,9 +85,13 @@ def update(self, predictor: Predictor) -> Predictor:
             self.restore(predictor.uid)
         predictor._archived = None
 
-        # We never exposed saving a model without training, so we should
-        # continue to do it automatically.
-        return self._train(predictor.uid)
+        # If the initial response is invalid, just return it
```

Comment:
We will never actually hit this branch until the API bug fix is in, but the logic is correct and tested for now.